pallets/automation-time/src/lib.rs

```diff
@@ -1344,19 +1355,34 @@ pub mod pallet {
 			.map_err(|_| Error::<T>::TimeSlotFull)
 		}
 
+		// TODO: move to a utils upon this MVP works
+		// We cannot use Rust std inside substrate runtime so i need to re-invent this
+		// Convert a decimal number to its string re-presentation where each character is a digit
+		// of the original number
```

Comment:
hmm it’s strange that the code base lacks of basic utils like this. Let’s find a home for this util function in this PR. @imstar15 where do we currently store utility functions?